### PR TITLE
Website: Load babylon parser for json formatting

### LINKED ISSUE #2606: JSON formatting not working in playground.
Here is the [repl link ](https://prettier.io/playground/#%7B%22content%22%3A%22%7B%5Cn%20%20%5C%22hello%5C%22%3A%20%5B%20%7B%5Cn%20%20%20%20%20%20%5C%22_id%5C%22%3A%20%5C%225988946e45e52d60b33a25c7%5C%22%2C%5Cn%20%20%20%20%20%20%5C%22latitude%5C%22%3A%2050.087977%2C%5Cn%20%20%20%20%20%20%5C%22longitude%5C%22%3A%2072.167197%2C%5Cn%20%20%20%20%20%20%5C%22tags%5C%22%3A%20%5B%5Cn%20%20%20%20%20%20%20%20%5C%22nulla%5C%22%2C%5Cn%20%20%20%20%20%20%20%20%5C%22ullamco%5C%22%5Cn%20%20%20%20%20%20%5D%2C%5Cn%20%20%20%20%20%20%5C%22friends%5C%22%3A%20%5B%5Cn%20%20%20%20%20%20%20%20%7B%5Cn%20%20%20%20%20%20%20%20%20%20%5C%22id%5C%22%3A%200%2C%5Cn%20%20%20%20%20%20%20%20%20%20%5C%22name%5C%22%3A%20%5C%22Robinson%20Woods%5C%22%5Cn%20%20%20%20%20%20%20%20%7D%2C%5Cn%20%20%20%20%20%20%20%20%7B%5Cn%20%20%20%20%20%20%20%20%20%20%5C%22id%5C%22%3A%201%2C%5Cn%20%20%20%20%20%20%20%20%20%20%5C%22name%5C%22%3A%20%5C%22Lottie%20Hogan%5C%22%5Cn%20%20%20%20%20%20%20%20%7D%5Cn%20%20%20%20%20%20%5D%5Cn%20%20%20%20%7D%2C%5Cn%20%20%20%20%7B%5Cn%20%20%20%20%20%20%5C%22_id%5C%22%3A%20%5C%225988946ef6090217857d7b0f%5C%22%2C%5Cn%20%20%20%20%20%20%5C%22latitude%5C%22%3A%2047.460772%2C%5Cn%20%20%20%20%20%20%5C%22longitude%5C%22%3A%2085.95137%2C%5Cn%20%20%20%20%20%20%5C%22tags%5C%22%3A%20%5B%5Cn%20%20%20%20%20%20%20%20%5C%22aliqua%5C%22%2C%5Cn%20%20%20%20%20%20%20%20%5C%22nulla%5C%22%5Cn%20%20%20%20%20%20%5D%2C%5Cn%20%20%20%20%20%20%5C%22friends%5C%22%3A%20%5B%5Cn%20%20%20%20%20%20%20%20%7B%5Cn%20%20%20%20%20%20%20%20%20%20%5C%22id%5C%22%3A%200%2C%5Cn%20%20%20%20%20%20%20%20%20%20%5C%22name%5C%22%3A%20%5C%22Mamie%20Wyatt%5C%22%5Cn%20%20%20%20%20%20%20%20%7D%2C%5Cn%20%20%20%20%20%20%20%20%7B%5Cn%20%20%20%20%20%20%20%20%20%20%5C%22id%5C%22%3A%201%2C%5Cn%20%20%20%20%20%20%20%20%20%20%5C%22name%5C%22%3A%20%5C%22Alejandra%20Mcdaniel%5C%22%5Cn%20%20%20%20%20%20%20%20%7D%5Cn%20%20%20%20%20%20%5D%5Cn%20%20%20%20%7D%5Cn%20%20%5D%5Cn%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D) 

Let me know if there is something I am missing.

Thanks for this awesome module.

### PR DESCRIPTION
**Pull request by azz**
Fixes #2606

### CODE REVIEW

File: website/static/worker.js [hunk 1]
```diff
@@ -22,9 +22,8 @@ var prettier = index; // eslint-disable-line
 var parsersLoaded = {};
 
 self.onmessage = function(message) {
-  const options = message.data.options || {
-    parser: "babylon"
-  };
+  var options = message.data.options || {};
```
Comment (azz) [inline website/static/worker.js]: This change is for backwards compatibility with _really_ old prettier links that didn't have `parser` set in options.

**Comment by josephfrazier** [comment]
Are the `./website/yarn.lock` changes just an artifact of rebuilding? They appear unrelated to the meaningful changes.

File: website/static/worker.js (hunk 1 above)
Comment (vjeux) [inline website/static/worker.js]: Thanks! The one I have in my autocomplete doens't have it and I need to tweak it manually all the time :P

**Comment by azz** [comment]
Yarn changes are from #2614 which didn't update the lockfile.

**Pull request merged**